src/components/alert/alert.tsx

```diff
@@ -46,6 +46,7 @@ import {
   LoadableComponent,
   componentLoaded
 } from "../../utils/loadable";
+import { MenuPlacement } from "../../utils/floating-ui";
```

Comment:
Note: I updated to use this shared resource instead of explicitly defining in `interfaces.ts`, since the accepted values are the same. That said, `alert` doesn't use `floating-ui`, so I can revert if needed.